tests/suites/os/tests/device-specific-tests/jetson-agx-orin-devkit/index.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+
+module.exports = {
+	deviceType: {
+		type: 'object',
+		required: ['slug'],
+		properties: {
+			slug: {
+				type: 'string',
+				const: 'jetson-agx-orin-devkit',
```

Comment:
@acostach  , what we can try here (if all the js files are the same) is to change

`const: 'jetson-agx-orin-devkit',`

to 

`enum: ['jetson-agx-orin-devkit-64gb', 'jetson-agx-orin-devkit', ....],`

where `...` is the list of the other device types. This will then run this test if the device type slug matches any of those in the list. 

In the js file, you can create a reference to the DT slug using `this.os.deviceType`

Reply:
thanks @rcooke-warwick , your suggestion worked very well, I updated the PR